### PR TITLE
buffs the automag and deagle by giving them their own bullet

### PR DESCRIPTION
**Pull request by Pariah919**
## About The Pull Request
45/15/3, staggers, slows, no shake however.
Consdering the automag has one second fire delay, it has a 1.2 dmaage mult for 54 damage.
Considering these guns fire incredibly slowly I don't think this is too much damage.

## Why It's Good For The Game
The automag and deagle sucks.

## Changelog
:cl:
balance: buffed automag and deagle to do much more damage, stagger and slow xenos on hit
/:cl:

### CODE REVIEW

**Comment by SplinterGP** [comment]
buff the automag more and reduce the deagle one, it has 1.2 damage mult and 0.7 fire delay, also remove knockback.

**Review by SplinterGP — APPROVED** [review]

File: code/modules/projectiles/guns/pistols.dm [hunk 1]
```diff
@@ -414,10 +413,10 @@
 
 /obj/item/weapon/gun/pistol/highpower
 	name = "\improper Highpower automag"
-	desc = "A Colonial Marshals issued, powerful semi-automatic pistol chambered in armor piercing 9mm caliber rounds. Used for centuries by law enforcement and criminals alike, recently recreated with this new model."
+	desc = "A, powerful semi-automatic pistol chambered in the devastating.50 AE caliber rounds. Used for centuries by law enforcement and criminals alike, recently recreated with this new model."
```
Comment (SplinterGP) [inline code/modules/projectiles/guns/pistols.dm:416]:
```suggestion
	desc = "A, powerful semi-automatic pistol chambered in the devastating .50 AE caliber rounds. Used for centuries by law enforcement and criminals alike, recently recreated with this new model."
```

**Review by SplinterGP — CHANGES_REQUESTED** [review]

File: code/modules/projectiles/guns/pistols.dm (hunk 1 above)
Comment (SplinterGP) [inline code/modules/projectiles/guns/pistols.dm]:
```suggestion
	desc = "A powerful semi-automatic pistol chambered in the devastating.50 AE caliber rounds. Used for centuries by law enforcement and criminals alike, recently recreated with this new model."
```

**Review by SplinterGP — CHANGES_REQUESTED** [review]

File: code/modules/projectiles/guns/pistols.dm [hunk 2]
```diff
@@ -414,10 +413,10 @@
 
 /obj/item/weapon/gun/pistol/highpower
 	name = "\improper Highpower automag"
-	desc = "A Colonial Marshals issued, powerful semi-automatic pistol chambered in armor piercing 9mm caliber rounds. Used for centuries by law enforcement and criminals alike, recently recreated with this new model."
+	desc = "A, powerful semi-automatic pistol chambered in the devastating .50 AE caliber rounds. Used for centuries by law enforcement and criminals alike, recently recreated with this new model."
```
Comment (Pariah919) [inline code/modules/projectiles/guns/pistols.dm:416]:
```suggestion
	desc = "A powerful semi-automatic pistol chambered in the devastating .50 AE caliber rounds. Used for centuries by law enforcement and criminals alike, recently recreated with this new model."
```

**Review by SplinterGP — APPROVED** [review]

**Review by TiviPlus — APPROVED** [review]

**Pull request merged**